src/watchkit.cs

```diff
@@ -772,7 +772,7 @@ interface WKAudioFilePlayerItem {
 		NSError Error { get; }
 
 		[Export ("currentTime")]
-		double CurrentTime { get; }
+		double CurrentTime { get; [Watch (3,2)] set; }
```

Comment:
Ugh, adding a setter is a breaking change (because the property is virtual, and code that overrode the property would be broken because it wouldn't override the setter).

We've added a `SetX` method for other similar cases.